File: Csharp/VueClient/Models/Enum/StatutCommande.cs
using NpgsqlTypes;

namespace VueClient.Models.Enum
{
    public enum StatutCommande
    {
        [PgName("EN_ATTENTE")]
        EN_ATTENTE,
        [PgName("ANNULEE")]
        ANNULEE,
        [PgName("TERMINEE")]
        TERMINEE,
        [PgName("VALIDEE")]
        VALIDEE
    }
}

File: Symfony/BrasilBurgerGest/src/Entity/Enum/StatutCommande.php
<?php
namespace App\Entity\Enum;
enum StatutCommande: string {
    case EN_ATTENTE = 'EN_ATTENTE';
    case ANNULEE = 'ANNULEE';
    case TERMINEE = 'TERMINEE';
    case VALIDEE = 'VALIDEE';
}